lib/src/connected_device_operation.dart

```diff
@@ -50,15 +49,15 @@ class ConnectedDeviceOperationImpl implements ConnectedDeviceOperation {
 
     return _controller
         .readCharacteristic(characteristic)
-        .asyncExpand((Object _) => specificCharacteristicValueStream)
+        .asyncExpand((Object? _) => specificCharacteristicValueStream)
```

Comment:
because of  the signature of asyncExpand `  Stream<E> asyncExpand<E>(Stream<E>? convert(T event))` . There is nothing I can think of that I can do about it

Reply:
> ... the signature of asyncExpand [is] `Stream<E> asyncExpand<E>(Stream<E>? convert(T event))`

My question is why `T` is optional (`Object?`)? Why `_control.readCharacteristic(·)` returns `Object?`? Does it make a lot of sense?